### ISSUE TITLE
Add Option to Disable Save Prompt on Exit

### ISSUE THREAD

**Issue by jayvzh** [marktext/marktext#3883]
Describe your feature request
I would like to request an option in Mark Text that allows users to disable the save prompt when exiting a markdown file in edit mode, even if no changes have been made. This feature would help improve the user experience by removing the unnecessary prompt, which can be disruptive during regular usage.

What problem does this feature solve?
Currently, Mark Text defaults to edit mode when opening a markdown file, and even if no changes are made, it prompts the user to save the file upon exiting. This behavior can be annoying and unnecessary, especially for users who prefer to exit without having to confirm saving every time.

Additional context
It would be helpful if there was a setting or toggle in the preferences to disable this behavior, so that users can exit without seeing the save confirmation dialog, even if no changes were made.
The option could be something like "Disable save prompt on exit when no changes are made."
This would provide more flexibility and a smoother experience when using Mark Text for quick edits.

**Comment by Not-Sure-Whats-Going-On** [comment]
Yes please.  Why this behavior?  When no changes have been made the Save prompt is frustrating and unnecessary.

**Comment by sunisstar** [comment]
support! and it seems that the author has abandon this project.
